cla_backend/apps/cla_eventlog/management/commands/find_and_delete_old_cases.py

```diff
@@ -22,7 +22,18 @@ def handle(self, *args, **kwargs):
         instance = FindAndDeleteCasesUsingCreationTime()
         cases = instance.get_eligible_cases()
         if len(args) and args[0] == "delete":
```

Comment:
`len(args)` and `args` will have the same truthiness, so no need to get the length here purely to check if it's non-zero, we can do `if args and args[0] == "delete"`

Reply:
Done